pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -150,6 +150,13 @@ public void updateDynamicConfiguration(@PathParam("configName") String configNam
         return BrokerService.getDynamicConfiguration();
     }
 
+    @GET
+    @Path("/configuration/runtime")
+    @ApiOperation(value = "Get all runtime configurations")
+    public List<String> getRuntimeConfiguration() {
+        return pulsar().getBrokerService().getRuntimeConfiguration();
```

Comment:
We should check for super-user permissions, since with authentication turned on you don't necessarily want to expose the broker conf to everyone.

Reply:
Changed, thanks!